test/e2e/specs/block-deletion.test.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Internal dependencies
+ */
+import {
+	clickBlockAppender,
+	getEditedPostContent,
+	newPost,
+	pressWithModifier,
+	META_KEY,
+} from '../support/utils';
+
+const addThreeParagraphsToNewPost = async () => {
+	await newPost();
+
+	// Add demo content
+	await clickBlockAppender();
+	await page.keyboard.type( 'First paragraph' );
+	await page.keyboard.press( 'Enter' );
+	await page.keyboard.type( 'Second paragraph' );
+	await page.keyboard.press( 'Enter' );
+};
+
+const clickOnBlockSettingsMenuItem = async ( buttonLabel ) => {
+	await expect( page ).toClick( '.editor-block-settings-menu__toggle' );
+	const itemButton = ( await page.$x( `//*[contains(@class, "editor-block-settings-menu__popover")]//button[contains(text(), '${ buttonLabel }')]` ) )[ 0 ];
+	await itemButton.click();
+};
+
+describe( 'block deletion -', () => {
+	describe( 'deleting the third block using the Remove Block menu item', () => {
+		beforeAll( addThreeParagraphsToNewPost );
+
+		it( 'results in two remaining blocks', async () => {
+			await clickOnBlockSettingsMenuItem( 'Remove Block' );
+			expect( await getEditedPostContent() ).toMatchSnapshot();
+		} );
+
+		it( 'positions the caret at the end of the second block as evidenced by typing additional text', async () => {
```

Comment:
An `it` test should be standalone, and not make any assumptions about other `it`s being run or the order in which they're run (for example, to enable parallelization of test execution). This seems like it should be part of the previous test case.

Reply:
Also FWIW it's fine for a single test case to contain multiple `toMatchSnapshot`